packages/dtslint/src/index.ts

```diff
@@ -26,6 +26,10 @@ async function main(): Promise<void> {
   let tsLocal: string | undefined;
 
   console.log(`dtslint@${require("../package.json").version}`);
+  if (args.length === 1 && args[0] === "types") {
```

Comment:
This is maybe not a mega related question, but _why_ do we require the first arg to be `types` at all?

Reply:
The args are just passed on to dtslint joined by `/` I think, so I don't think it's strictly required to be `types`. Not sure.